framework/scripts/wazuh_server.py

```diff
@@ -320,14 +316,7 @@ def get_script_arguments() -> argparse.Namespace:
     ####################################################################################################################
     start_parser.add_argument('-f', help='Run in foreground', action='store_true', dest='foreground')
     start_parser.add_argument('-r', help='Run as root', action='store_true', dest='root')
-    start_parser.add_argument(
-        '-c',
-        help='Configuration file to use',
-        type=str,
-        metavar='config',
-        dest='config_file',
-        default=common.OSSEC_CONF,
-    )
+    # TODO(26693) - Delete this parameter that isn't used
```

Comment:
This flag is no longer needed as explained in https://github.com/wazuh/wazuh/pull/26645#discussion_r1840932353

Reply:
Fixed in c24216b9677fbc2f0325ac805d258b4105f66c3d.